source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnSetRenderer.mm

```diff
@@ -47,6 +47,7 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
                                                                   parentStyle:[viewGroup style]
                                                                    hostConfig:acoConfig
                                                                     superview:viewGroup];
```

Comment:
Seems to be an unnecessary whitepace/newline change that can be reverted.

Reply:
the change is added by clang-format that format the source.